src/assets/server/rest/v1/docs/e-mobility-oas.json

```diff
@@ -956,6 +1512,43 @@
         }
       }
     },
+    "/api/transactions/status/unassigned/count": {
```

Comment:
Weird this one :S
Where is it used?

Reply:
It is the "UNASSIGNED_TRANSACTIONS_COUNT" action, calling the "handleGetUnassignedTransactionsCount" method in TransactionService
After checking, it seems not to be used neither in dashboard or mobile app :)

Reply:
Okay, you can remove it then :)

Reply:
My bad, it used in the Tag components of the Dashboard :)

Reply:
Okay, we keep it then.